rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+---
+feature: community-team
+start-date: 2021-07-31
+author: Irenes (Irene Knapp)
+co-authors: ashkitten
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Establish an official community team to make moderation decisions, model social norms, and mediate interpersonal problems.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not sustainable to have to call on Graham any time there's a spammer or conflict that requires moderation, and we'd like to help the community become more self-regulating.
+
+# Goals
+[goals]: #goals
+
+* Model and enforce social norms
+* Mediate disputes when doing so is beneficial to the community
+* Respect the privacy of community members
+* Foster newcomers and develop their leadership potential
+* Moderate all places where community members interact, including Discourse, Matrix, and GitHub
+* Ensure that collaboration on shared technical goals is possible
+* Ensure that the Nix community is a welcoming space for people from marginalized backgrounds
+* Do not allow the Nix community to be a place for spreading ideas rooted in fascism or bigotry
```

Comment:
I have a feeling that fascism is a weirdly narrow specification here…

Reply:
How would you broaden it?

Reply:
What exactly are you trying to outline here (which you don't also put under bigotry/discrimination)?

Reply:
By fascism I mean: advocating against democracy and representative government (in society at large, not in NixOS specifically); advocating for an increase in state violence; advocating for the extermination of marginalized groups; things like that.

This is not intended to exclude political ideas in general, whether or not I personally agree with them. It is setting a very narrow goal which I would hope that everyone can agree on, not even the goal to *fight* facism but simply to not allow the Nix community to be used to *promote* it.

Reply:
I agree with @7c6f434c There are too many things to “not allow” to effectively enumerate. I’m not sure what this adds. Simpler to remove this bullet point.

Reply:
I think that there is a very real danger, for NixOS as for every technical community everywhere, of people trying to use our platform to spread fascism and bigotry. For an example of bigotry, take a look at the collapsed comment further down, the one which is a transmisic attack on me as an individual (TW for transmisia). I think failing to make it a goal to keep that stuff out of Nix would be a mistake.

Also, it's part of the value of "advocacy" that "We don't simply ignore attacks on our values, we speak up and take action to defend them."

The purpose of the "goals" section overall is to set a bar by which the rest of the document can be evaluated. It should be regarded as informative, rather than normative, but it's important to have it because it shapes the discussion.

Reply:
I think as a statement of principles, it is useful to declare a lack of tolerance for bigotry and fascism within the community. Communities which do not follow such principles sometimes develop a problem wherein people spreading bigoted or fascistic ideas alienate other members of the community and drive them away.

This goes hand-in-hand with the previous point of creating a welcoming space for people from marginalized backgrounds, but is useful to state explicitly, because a space that is welcoming to people from marginalized backgrounds cannot be a space that is welcoming to fascists.

Reply:
First, what you describe is not that well-aligned with specifically fascism. Second, an increase in enforcement of, say, food safety labelling laws usually comes with strictly speaking an increase in state violence. Third, in case of two countries considering exterminating the entire population of each other, does «the entire population of a country» count as a marginalised group?

I think using a word that has an actual definition (or a handful, with different scope…) among those who study the thing, then a few different colloquial definitions, then a few over-broad «everything bad» definitions is more confusing than useful. If you want to explicitly include calls to initiation of violence, well, maybe (it is not that all initiation of violence is wrong, it is that some places are unsuitable for its discussion). Using unspecified version of the notion of fascism looks a bit of «how to accidentally build a language barrier».

Reply:
I think specificity is important. It signals to fascists that they can't get away with "You're violating my right to support fascism"-type arguments. It signals to the rest of us that if we report someone being fascist, then we're less likely to get a "Well, they were just exercising their right to free speech"-type counterarguments.

Reply:
> Communities which do not follow such principles sometimes develop a problem wherein people spreading bigoted or fascistic ideas alienate other members of the community and drive them away.

If you already condemn bigotry and discrimination, and maybe promotion of initiation of violence, are there many of those who you would call spreading fascistic ideas who would not fail these tests? (And if yes, are you sure your definition is a good idea?)

> because a space that is welcoming to people from marginalized backgrounds cannot be a space that is welcoming to fascists.

Erm, I would like to ask you to be more careful with wording. 

Sure we don't want to welcome any promotion of oppressive actions (hopefully regardless of the details of the promoted ideology), but if you try to actively check everyone for being a fascist who just uses a seemingly unrelated set of accounts to discuss politics — that will end up more destructively than some level of «do not ask, do not tell, do not come under a nickname which promotes violence before you even say anything».

As far as I remember, historical fascism was quite glad to recruit exactly the people from some of the most typical marginalised backgrounds in the relevant countries (although not all of them), but that's a different story.

Reply:
Perhaps something simpler and with a more positive bent? "The Nix community is a place for spreading ideas about Nix". Not sure if adding "and nothing else." is even necessary. My thought process is that it is easier to precisely describe what something is than trying to do it by negation. Maybe say "Nix related technology", but even that seems to be superfluous.

Reply:
I think the idea is that some things should not be said even in Nix-Offtopic or whatever it is called.

Reply:
Hi, I'm just catching up on this, my apologies. I am not trying to "actively check everyone for being a fascist". I think the content of the RFC goes into considerable detail on what we are actually proposing.

I gave such a specific definition of fascism in terms of democracy, violent elimination of minorities, etc because I was attempting to make it very clear that I am not referencing any position for or against "food safety labelling laws" or anything of the sort. I encourage anyone in doubt on this point to re-read my earlier comment.

Reply:
About investigating alternative nicks — that was a comment on wording in a quote not from you.

My claim is that the modern use of the word fascism is just not consistent enough (globally) to consider it a single word. I am not sure from your message if you want to avoid promotion of either of the mentioned things or of the combination. In the former case these are vague enough that even if written out those will be understood non-uniformly. In the latter case — even if initiation of violence is not always wrong, maybe it is simpler to just declare all NixOS spaces, even offtopic ones, unsuitable venue?

Reply:
It's either. There is fascism that is not obviously linked to bigotry, and there's bigotry that is not obviously linked to fascism, and our goal is to make Nix spaces not be suitable places for spreading either of them.

I agree that there's a lot of ways the word gets used, but I've found that people who are spreading those ideas reliably know that "fascism" and "bigotry" are in fact talking about the ideas they're spreading. The main point of having it in the "goals" section is to inform discussion about the rest of the document, and I think it's already accomplished that purpose.

Reply:
> If you already condemn bigotry and discrimination, and maybe promotion of initiation of violence, are there many of those who you would call spreading fascistic ideas who would not fail these tests? (And if yes, are you sure your definition is a good idea?)

Yes, the point I was trying to get across was that *some* statement of condemnation of bigotry and discrimination is useful to have. To go more in depth on that, I believe that in principle condemnation of bigotry should be enough, although ideally we also don't want to sound too vague so as to sound meaningless. I don't have any specific wording suggestions here.

> Erm, I would like to ask you to be more careful with wording.
>
> Sure we don't want to welcome any promotion of oppressive actions (hopefully regardless of the details of the promoted ideology), but if you try to actively check everyone for being a fascist who just uses a seemingly unrelated set of accounts to discuss politics — that will end up more destructively than some level of «do not ask, do not tell, do not come under a nickname which promotes violence before you even say anything».
>
> As far as I remember, historical fascism was quite glad to recruit exactly the people from some of the most typical marginalised backgrounds in the relevant countries (although not all of them), but that's a different story.

I don't think "not being welcoming to fascists" requires us to start hunts for fascists. Rather, fascism, or (if you want to be more precise and more broad at the same time) bigotry cannot be generally tolerated within a community, or your community with accrue people who are bigoted, and exclude people who are targets of that bigotry.

Reply:
Yep - both of you are correct, and thank you. We're trying to focus on bad *behavior*, not on bad *people*, because people can change. And in fact we encourage people to change and that's a huge part of what these values are about.

Reply:
My argument, for the record, is not even that people can _change_ as people, but that people (approximately always) choose not to express some parts of their personality in some contexts. Negotiating this is much easier than inducing deep change or something.

Reply:
The Nix community and the associated forums/spaces/etc should be about Nix. I think that is the message that we should convey. We should not have to elaborate the various things it is not for. Specifically calling out a few horrible concepts and then not mentioning other horrible ones (sadly the world is full of too many to enumerate) is "weirdly narrow" as @7c6f434c put it.

Even though "Nix-offtopic" has a much wider realm of acceptable discussion topics than just Nix, it is still seems to be generally centered around Nix and the interests and activities of people involved in Nix. Consider if it became a regular collaboration space for some online gaming community to coordinate tactics -- clearly, at some point they'd be told it's not appropriate to take over a the chat room in that manner and that they should find another venue for their discussion.

Reply:
Thanks for clarifying. Yes. I'm in full agreement with that.

Reply:
> The Nix community and the associated forums/spaces/etc should be about Nix. I think that is the message that we should convey. We should not have to elaborate the various things it is not for. Specifically calling out a few horrible concepts and then not mentioning other horrible ones (sadly the world is full of too many to enumerate) is "weirdly narrow" as @7c6f434c put it.
> 
> Even though "Nix-offtopic" has a much wider realm of acceptable discussion topics than just Nix, it is still seems to be generally centered around Nix and the interests and activities of people involved in Nix. Consider if it became a regular collaboration space for some online gaming community to coordinate tactics -- clearly, at some point they'd be told it's not appropriate to take over a the chat room in that manner and that they should find another venue for their discussion.

We're enumerating values that it *is* for, in the values section, which is normative (sets forth things that will apply going forward). We're enumerating these two very specific things that it is *not* for, fascism and bigotry, in the goals section, which is informative (intended to help in evaluating whether the document is doing its job, and in this discussion). I think that, in a world where fascism and bigotry are on the rise and are actively spreading through communities of all kinds including technical communities, it is important to make a statement against them.

If you would like to add something to the mention of fascism and bigotry in the goals section, I would love to hear what. I tried pretty hard to think of anything else that was a clear and present danger to technical communities, and I couldn't think of anything that wasn't captured by either "fascism" or "bigotry". I wouldn't support adding, for example, "child starvation" to this section because, while child starvation is horrible, it is not something which is actively spreading through technical communities.

Reply:
Including fascism by (by now effectively meaningless as a means of communication) name and not by includion in bigotry and discrimination (and maybe initiation of violence) invites rule-lawyering what the hell fascism is (are you allowed to say anything positive about Singapore ever? there are people who make an argument the society structure there is kind of fascist…) Which, as another side of the same coin, seems to also be a problem in some technical communities…

I am not sure I have naturally seen enough cases of propagation of exterminate-minorities kind of fascism to say I know what are the other popular violently destructive ideas are.…

Reply:
"Fascism" is term used by leftists to attempt to divide their political opponents, just as accusations of "Marxism" by the right attempts to divide the left. To me, they have the same meaning. I suggest we either include with the term "Fascism," "Marxism," or exclude them both.

Reply:
The inclusion of fascism in the "goals" section cannot invite rules-lawyering because it is not creating a rule. The purpose of the goals system is to guide this discussion which we are having. I think it's doing that.

We don't need to define fascism precisely, because if people whose goal is actually to spread fascism recognize that it's about them, the purpose of including it has been accomplished.

Reply:
> "Fascism" is term used by leftists to attempt to divide their political opponents

I am sorry you feel this way, but I do not sympathize with it. Everything in this RFC is neutral to political ideology, except to the extent that said ideology may conflict with the statement of values.

Furthermore, in the authority section of the RFC, we explicitly state that "Team members are not expected to be impartial in the sense of pretending not to have personal beliefs or goals, but are expected to hold the community's values above their own, in their official actions."

I think this sets a very high bar for keeping partisan politics out of the community team's duties, and is everything that anyone could ask for in that regard.

Reply:
I have a BS in Political Science and did my best to immerse myself in the practical field while studying it. Every leftist activist rally I have observed then and since winds up in a one-sided shouting match calling conservatives and moderates who question them, "fascists." It's undeniable. [This is a sort of dog-whistle for leftists and Marxists](https://www.youtube.com/watch?v=KLdDmeyMJls). 

I think you're accomplishing nothing more than attempting to divide us and waste our energy.

Reply:
Please let this document live up to it's claims: "principles over rules".

Reply:
> I think you're accomplishing nothing more than attempting to divide us and waste our energy.

Your feedback is noted and appreciated. It's regrettable that there are people who believe that, but unavoidable. I nonetheless remain committed to demonstrating good faith for anyone who wants to engage with the ideas in the document.

Reply:
> We don't need to define fascism precisely, because if people whose goal is actually to spread fascism recognize that it's about them, the purpose of including it has been accomplished.

Wait what? By that principle you can replace it with «being biologically human». People who spread fascism will definitely recognise it as about them. A description should clearly cover the central example of what you meant (don't bigotry and incitement of violence already do?), clearly not cover what you did not actually mean, and have, erm, comparable scope when interpreted by different people.

> The inclusion of fascism in the "goals" section cannot invite rules-lawyering because it is not creating a rule.

Writing «please do not rule-lawyer» does not prevent the activity, it _only_ gives the community team the explicit permission to ignore its conclusion (they do have an implicit permission anyway).     

> Your feedback is noted and appreciated. It's regrettable that there are people who believe that, but unavoidable.

It is avoidable by using less sloppy writing. And by covering the things you want to avoid in a way not stuck in its exact manifestations of today. Frankly, from _this_ thread I would have questioned your good faith by now if I had not seen you perform moderation work elsewhere.

Reply:
Once again: How to handle a concrete instance of anti-vaccination ideology would be a moderation decision, which would be made by the community team with reference to the statement of values. The wording in the goals section, which your suggested change is to, is here for the purpose of guiding discussion about the rest of the RFC; it does not create any rules which would be applied once the RFC is finalized.

For the sake of answering your question anyway, I don't think there's a credible case that a mere opposition to vaccination is rooted in fascism or bigotry, unless it's justified by reference to antisemitism or something of the sort. I don't think the RFC needs to take a stance on vaccination.

"Hostile ideas" seems like it would include, really, any idea that anyone believes strongly in. We don't want to do that. We want the Nix community to be a place where people can discuss ideas (especially ideas about Nix). We just don't want it to be a place for spreading fascism or bigotry.

Reply:
So, have you ever seen an example of fascism not also being an obvious (also to the spreader, if they happen to be human and to read what they write, as neither is strictly speaking guaranteed) example of bigotry?

Reply:
> Do not allow the Nix community to be a place for spreading ideas rooted in fascism or bigotry

I agree that the wording here is clumsy. *Fascism* stands out as being very out of place in a document that otherwise tries to stay as uncontroversial as possible. I fully agree with the intent, but there must be another way to express it. I like @7c6f434c 's line of reasoning that we should either list a bunch of things we don't want to allow, or just remove this altogether and leave bigotry (apologies if I've summarized the position incorrectly, the threads here have become too long to follow thoroughly).

At the risk of exchanging one vague term for another, can't we just say "violence or bigotry"?

Also keeping in mind that we're not writing a political or philosophical manifesto here. What classifies as *violence* is technically as open-ended as with fascism, but I think that for our purposes, it's good enough.

Reply:
> So, have you ever seen an example of fascism not also being an obvious (also to the spreader, if they happen to be human and to read what they write, as neither is strictly speaking guaranteed) example of bigotry?

There's a brand of advocacy for fascism which appeals heavily to science. I think that, certainly, the people who came up with those talking points understand that the science doesn't actually support them and that it's actually rooted in bigotry, but I also think it's quite common for people to repeat the talking points while sincerely believing them. After all, not everybody who reads a thing and decides that they agree with it takes the time to actually look up relevant research, especially if the bulk of the peer reviewed research on the topic is written by people they don't feel politically aligned with.

So: Yes, I've seen such examples, and I can't even be completely sure which ones they were, since I can't look inside people's hearts.

Reply:
> I agree that the wording here is clumsy. _Fascism_ stands out as being very out of place in a document that otherwise tries to stay as uncontroversial as possible. I fully agree with the intent, but there must be another way to express it. I like @7c6f434c 's line of reasoning that we should either list a bunch of things we don't want to allow, or just remove this altogether and leave bigotry (apologies if I've summarized the position incorrectly, the threads here have become too long to follow thoroughly).

My deepest sympathies on the length of the threads. I expected this, and it's fine, it's what community participation looks like, but it is a lot to keep up with! I'm glad you weighed in despite the challenges.

I believe that any way to express it that actually says anything, will be equally contentious. It's out of place because we very carefully kept the necessary controversy down to just the two words "fascism" and "bigotry". Both of those words have provoked considerable discussion already, so I'm afraid I can't agree that "bigotry" is less controversial than "fascism" or that we'd gain anything by removing only "fascism".

With that said, in an out-of-band conversation with @blaggacao yesterday, we discussed the idea of factoring these two words out into a separate list that's maintained by the community team. The words would still appear in the RFC, in that they would be given as the initial contents of that list.

I admit that I don't see the need for such a factoring-out, but I mention it because it seems aligned with your suggestion, and because I had previously said I'd share any ideas that came out of that out-of-band discussion with the thread.

The reason I don't think factoring it out makes sense is the same as the reason I'd previously given for why I don't think expanding it makes sense, but I can't immediately find where I said it, so I'll say it again.

I've put a lot of thought into this, and I can't think of anything else that would go in that list. Neither fascism nor bigotry is a new idea; on the contrary, they predate recorded history. Human civilization may invent fifty new types of computer every year, but I don't think we have to worry about anyone inventing something horrifying that's the equal of fascism or bigotry in its scope and danger. Nearly every new horrifying thing I see falls into one or both of those existing categories.

> 

> 
> At the risk of exchanging one vague term for another, can't we just say "violence or bigotry"?
> 
> Also keeping in mind that we're not writing a political or philosophical manifesto here. What classifies as _violence_ is technically as open-ended as with fascism, but I think that for our purposes, it's good enough.

Unfortunately, what counts as "violence" is highly political and I can think of many very contentious fights over it. For example, many people would say that mass arrests are not a form of violence because it's being done (nominally) in the service of the law. Mass arrests, when they're motivated by the desire to suppress political dissent, are very clearly included in "fascism". So I do think we'd lose something by making that change, and we wouldn't gain anything.

Reply:
> anyone inventing something horrifying that's the equal of fascism or bigotry in its scope and danger.

If you qualify early CPC China «cultural revolution» as «fascism», you stretch the meaning of the words so far they . If you say that the defining example of «fascism» (1930's Italy) is _clearly_ more horrifying than «cultural revolution» as implemented, I find it hard to trust your judgement on writing norms.

Of course I do not expect you literally believe the latter; I cannot say if you believe the former. 

> Mass arrests, when they're motivated by the desire to suppress political dissent, are very clearly included in "fascism".

Note that this clarification doesn't really help much; in the sense that right now there are peaceful outdoor protests against mask laws that end up in mass arrests (usually short-term) due to breaking the same mask laws. It's a mess enough without needing to wonder whether it is also fascism.

Reply:
I agree with @7c6f434c. If your notion of fascism stretches beyond recorded history, then we're not talking about the same thing at all, and that in turn reduces my trust in the intentions behind this whole RFC..

Who gets to decide what ideas are rooted in @ireneKnapp's/@ashkitten's definition of fascism?


I would invite us to stick to the common definition of terms here, rather than our own personal ones: fascism has a very specific historical connotation, rooted in time and space. If we mean "institutional violence", "mass incarceration" or "persecution of minorities", then we should mention those.

> many people would say that mass arrests are not a form of violence because it's being done (nominally) in the service of the law. Mass arrests, when they're motivated by the desire to suppress political dissent, are very clearly included in "fascism"

This is great, because I specifically do not want people banned for "spreading" (which I guess should more honestly be changed to "sharing") their approval for mass incarceration to suppress political dissent. If I'm looking for people to agree with politically, it's not in the NixOS community chat's that I'll go first, thanks. 

Honestly, I just find it weird to even be discussing fascism in a NixOS RFC.

Reply:
There's a very good reason we only said "fascism" in the goals, and not in the values. It's difficult to operationalize, just as you're both saying. That is, it's hard to say what is in or out, in practice. You don't have to trust how we define it, because its presence in the goals has no effect other than to guide this discussion. I hope you don't feel that mentioning fascism in the goals has prevented you from discussing ideas on this thread for fear they might be called fascism. If you do, please say something.

I'm not a historian, and not qualified to really back up my belief that the ideas behind fascism as it's understood from 20th-century and 21st-century examples are not new ones. I don't think it's really that important a point and I won't try to defend it. I was just hoping to add some color to help people understand what we mean. It seems to have had the opposite effect, which is regrettable but I guess it's better to have it out of the way now.

As far as what *will* matter going forward, it's my belief and, I think, @ashkitten's as well that the statement of values in its present form is sufficient to achieve the goal of keeping fascism from spreading through the Nix community. There's nothing hidden there, you don't have to guess what the statement of values means. It's very explicit and puts all the cards on the table. You're welcome to disagree with the statement of values, and we should absolutely talk about those disagreements, but the whole point of it was that nobody has to wonder what values the community team would enforce.

Reply:
> By fascism I mean: advocating against democracy and representative government (in society at large, not in NixOS specifically); advocating for an increase in state violence; advocating for the extermination of marginalized groups; things like that.

Again, I've got a four-year degree in Political Science, during which I learned that democracy is defined by some political theorists as tyranny of the majority over the minority, and that a representative government also has its criticisms, such as corruption by special interests.

I also learned that the state has a near monopoly on legitimate use of force. If district attorneys refuse to prosecute and police do not enforce the law, citizens may feel forced to take the law into their own hands.

The United States reserves the death penalty for particularly egregious crimes, some states and many other nations prohibit it altogether, and some may argue marginalized groups are most affected, but others may argue that it is a necessary deterrent and even required lest lynch mobs begin making their own justice. And the debate is still strong in the United States.

If I engage in political debates on democracy at my local University, argue amongst my technical friends and acquaintances on Twitter for the enforcement of our laws, or write a letter to the editor of my local newspaper advocating the death penalty, am I a fascist, by the authors' definition?

Would I be prohibited from engaging my duty as a citizen to exercise my rights to speech, if I wish to be a part of this community? What other "things like that" are going to restrict my rights?

My favorite socialist writer, [George Orwell, wrote, in 1947](https://orwell.ru/library/articles/As_I_Please/english/efasc) that "'fascism' is meaningless" and "Except for the relatively small number of Fascist sympathizers, almost any English person would accept ‘bully’ as a synonym for ‘Fascist’. That is about as near to a definition as this much-abused word has come." So it would seem this is just a label for anyone we might dislike.

But a quick perusal of wikipedia and dictionaries gives us a few definitions of fascism (pro-labor, pro-socialism, pro-regimentation of society), thus it would seem to me that I could best be defined as "anti-fascist" since I do not prefer collective bargaining, expansive government, or regimentation of our society.

If we apply this historic or common standard to the authors of this document, how would they fare?

Reply:
@aaronchall: We are not advocating the death penalty for people we object to.  We only seek to exclude them from our community.  Unlike the death penalty, people excluded from our community are not denied any freedoms.

Reply:
@Gabriel439 are you saying you object to someone being a member of our community based on their mainstream politics being different from yours?

Reply:
The number one rule I live by when it comes to the internet is "don't drag politics into conversation". That does NOT mean that I can't talk about politics with someone, it means that I won't mention politics while talking about other stuff. Politics talk has to be completely consensual on both sides. Saying that some some political views are prohibited is against that rule. I don't give a rats ass about someone being a fascist, hell even a cold blooded murdered would be a-ok. Its their own damn thing, none of my concern as long as they write good code, i like talking to them and they don't bring it up I dont care.

Reply:
While I completely grock the idea of not caring about the politics of our community members, I also understand the stance that I don't really want to work with bigots or political extremists (like tankie communists), even if they are good at tech.

My primary problem with this document is that the authors have gotten consistent feedback to remove their vague, ambiguous, or nebulous reference to politics (that is, the term "fascism") from this document, and they have failed to respond to this feedback. I have done my best to outline the problem, as I see it, using reasonable counterexamples that could be used abusively. 

Additionally, as I have said before, this document is far too long. Based on the lack of cooperativeness of the authors and the length of the document we need completely new authorship of any kinds of foundational or governing documents. By that, I mean completely start over. And I would prefer the authors to be members of the core team.

Reply:
I agree with that, start over, I'd be much happier with trusting someone with core who has a real stake in the community. Just don't drag politics into it. I'd start with "be a decent human being" and expand on it so that people get what it means to be nice. That's it, simple, universal and not full of vagueness.

Reply:
> are you saying you object to someone being a member of our community based on their mainstream politics being different from yours?

@aaronchall: I support excluding people who promote bigotry and fascism, regardless of whether bigotry and fascism are mainstream political beliefs and also regardless of whether those beliefs differ from my own.  Perhaps those criteria would exclude me, either now or in the future, but I believe the criteria are good because I believe that fascism and bigotry should be unwelcome within our community

Reply:
There is a difference in believing in something and stating that when asked and unsolicited promotion, while I condemn the latter, I support the former. Everyone should have the right to promote their views when asked, perhaps not directly in public, ie they should be told to take to to DMs, but they shouldn't be judged by it. Of course if a racist shows up and says: "im not working with that person because theyre black" it counts as unsolicited promotion. My viewpoint may be extremely apathetic, I believe it's the most calm and inclusive(even though i dislike this word) way to handle things. Just dont touch with even with a 20 foot pole and no one can hold you accountable. Everyone's happy, at least those that dont have malicious intent.

Reply:
@MagicRB: I get where you're coming from.  I used to think like that, too.  The thing that changed my mind was to recognize that the problem with fascism and bigotry isn't that they *say* harmful things, but rather that they *do* harmful things in ways that often have plausible deniability because they don't have to publicly disclose the motives for their actions.  This is especially true if they have positions with formal authority.  Sometimes the only way to establish malicious intent is to reference their pattern of behavior outside the community.

Reply:
Well I'd apply different rules to people in power, those positions I would scrutinize every aspect of that person to determine what effect might their rule have, but they shouldn't be excluded from the community. But yeah, I see where you're coming from, I think there is some merit to both our opinions. Luckily I'm not the one with the final say, I don't believe I'd make the right call in the end, apathy at it's finest lol.

Reply:
Hi all, 

The feedback about the vagueness of "fascism" has been heard, and discussed within the RFC shepherd team. We have been working on a forthcoming re-draft of RFC 98 which we believe will address this and the other concerns that came up during the initial flurry of discussion here on GitHub. We ask you all to be patient and wait for the next draft, which will be substantially re-worked. The RFC process is a slow one; everyone involved is an unpaid volunteer, and producing polished writing takes time.

I'm sorry that we haven't been having those conversations in view of the public. It was the opinion of the entire shepherd team that it would be counterproductive to encourage further back-and-forth here on the GitHub thread. It is my personal opinion that GitHub is almost the worst possible platform for this kind of discussion, as it's all but impossible for anyone coming into a discussion late to catch up on what's been said.

Please feel free to start Discourse threads with specific concerns about RFC 98, or to come to the Matrix chat room which has been set aside for engagement between the authors, the shepherd team, and the rest of the community.

I will not be replying directly to Aaron's remarks; in a previous discussion elsewhere on this page, he asked me not to, and I am respecting that. I'm replying here for the benefit of everyone else on the thread.

Thanks!

Reply:
What I recall that I said, for the *benefit* of your rfc, *specifically* that I suggest you "stop responding to every critic" (citing ashkitten's comment since I cannot find mine) The reason for this is that your overactive posting on behalf of yourself leaves little room for anyone else to write anything in response to critics of your rfc - and there *hasn't* been much of anyone else addressing the criticisms, **especially from members of the core team.** So instead this looks like a personal pet project of yours that you are ramming down our throats.

Perhaps the thread has been removed, but I can no longer find one of your final responses accusing me of some kind of gender related bigotry. This is not personal, but the only thing I know of you is your labor activism getting you fired, and as you may have gathered I (and others) associate labor activism with marxism/fascism. Both my wife and I have been union members, but we have not had good experiences from it. Perhaps if I knew you to disavow Marx I would be less critical, but instead you only seem to be confirming my suspicions that you are a trained Marxist activist here to disrupt our community. I propose, in the spirit of deescalation, that you disavow all of Marx.

Also I posted:

> I suggest we vote. Thumbs down if you think we should drop terms like "fascism" and "communism". Thumbs up if you think we should include them. Respond in defense of communism if you think we should mention "fascism" and not "communism."

This had 5 thumbs down, no thumbs up, and no responses agreeing with the current state of this rfc mentioning "fascism" and not "communism". It seems we don't want politics in any governing document of ours.

Trouble finding posts, length, verbosity, the back and forth... the discussion on this document is now a complete mess. At some point you said:

> My deepest sympathies on the length of the threads. I expected this, and it's fine, it's what community participation looks like, but it is a lot to keep up with!

No, it's not what community participation looks like, it's what stinking pile of controversy looks like, and yes it seems to be exactly what you wanted.

I have now given this some more thought. I suggest any governing document to spell out that we will not accept bigotry on the basis of the following legally protected categories for employment (at least in the US, perhaps we can internationalize this?)

- race, 
- color, 
- national origin, 
- religion, 
- sex, 
- age, or 
- disability

Note that "fascism" isn't protected here, nor is it sanctioned. If by fascism you mean tankie marxists, nobody likes those types, and you have nothing to worry about. If by fascism you mean "bullying" as George Orwell said was the primary meaning given to the term by average folk, then you'll find we all hate bullying quite a lot - all of this controversy could have been avoided if you had just said what you meant. If you mean "anyone who disagrees with me" as the leftist activists do, then you'll find much disagreement from many quarters.

Don't feel like you have to respond. Perhaps a core committer will respond to my criticisms instead. I only have around a dozen or so posts on this rfc. Maybe others should take a turn.

Reply:
> spirit of deescalation

@aaronchall I feel like you are the one escalating things. Irenes said you were heard and they are addressing it. Please hold off on this discussion until they release their changes.

Reply:
Thanks Ryan, much appreciated.

I do need to clarify that I have not accused Google of firing me. That would be trivially disprovable, and for me to say it would be potentially defamatory. I have accused them of retaliating against me.

I have nothing else to add at this time.

Reply:
Nothing else to add, Including no disavowals of Marx?

@ryantm I hear you on holding off on the discussion. To demonstrate the hearing, I'll hold off now. But I certainly don't feel heard in the slightest. Glibly asserting the hearing and actually demonstrating it are two different things.

Reply:
I have a personal ethical belief that prohibits me from taking a strong stance on any book I haven't read.

Reply:
Can we please ban @aaronchall from this RFC? They have repeatedly shown that they are not interested in a constructive discussion, but in escalating things and pushing their own thematic agenda. They have been called out multiple times on this and keep doing it again and again. This is derailing the discussion, and it has to stop.

Actually, I'd be open to locking this RFC again until the feedback has been incorporated. Alternatively, maybe simply close it and re-open a new one so that the discussion may start fresh in the future.

Reply:
He may be very vocal about hating communist, but I see no reason to exclude him from the discussion. He does have a point that specifically naming fascism but not any of the other harmful ideologies is not excellent. But my vote isn't worth much in the yes of those that don't frequent the discord server ig

Reply:
> pushing their own thematic agenda

I fail to see how this is contrary to the purpose of the RFC process.

> Actually, I'd be open to locking this RFC again until the feedback has been incorporated.

I tend to be disappointed by RFC-discussion-locking, but here it would indeed make sense. Locked to be unlocked once new version appears; makes sense when the new version will appear only after some time and is expected to be quite different.

> Alternatively, maybe simply close it and re-open a new one so that the discussion may start fresh in the future.

Unsure if it will mean even more repetition of the same things in the parts that will stay similar to the current version. Maybe there is the benefit of getting one more week of discussion until we completely break GitHub UI again; not sure if it is worth ambiguity about what things are parts or not parts of the discussion.

Reply:
I want to remind people that we have a Discourse - https://discourse.nixos.org/ - which is a much better venue than GitHub for just about any sort of long-form discussion. It's not a chat room, it's more like a mailing list. Unlike GitHub, it's possible to actually keep track of what's being said there, even for people who don't want to spend every spare minute at the keyboard.